pyanaconda/threading.py

```diff
@@ -45,6 +45,9 @@ def __init__(self):
         self._objs_lock = threading.RLock()
         self._errors = {}
         self._errors_lock = threading.RLock()
+        self._main_thread = threading.main_thread()
+
+    def set_current_thread_as_main(self):
```

Comment:
I don't see a point in this. Why we want to have something which will be replaced immediately after import. 

Right now basically you will do:
```
self._main_thread = threading.main_thread()
self._main_thread = threading.current_thread()
```

Then you can remove the `initThreading()` completely and we don't need to call it at all. On the other hand I would leave this method here (but docstring should be here). It could be valuable to have the method to set current thread as main somewhere.

Reply:
The initThreading() should be called only from anaconda.py now, it is not called from the modules. I wasn't sure if main_thread is same as current_thread in anaconda.py. But yes, initThreading could be removed.